res/locales/fr_FR.po

```diff
@@ -505,68 +475,55 @@ msgstr ""
 "\n"
 "    – Deux normales\n"
 
-#: constraint.cpp:663
+#: constraint.cpp:685
 msgid "Must select an angle constraint."
 msgstr "Vous devez sélectionner une contrainte d'angle."
 
-#: constraint.cpp:676
+#: constraint.cpp:698
 msgid "Must select a constraint with associated label."
 msgstr "Vous devez sélectionner une contrainte avec une étiquette associée."
 
-#: constraint.cpp:687
+#: constraint.cpp:721
 msgid ""
 "Bad selection for angle constraint. This constraint can apply to:\n"
 "\n"
+"Angle between:\n"
 "    * two line segments\n"
 "    * a line segment and a normal\n"
 "    * two normals\n"
-msgstr ""
-"Mauvaise sélection pour une contrainte d'angle. Cette contrainte peut "
-"s'appliquer à :\n"
 "\n"
-"    – Deux segments de ligne\n"
-"    – Un segment de ligne et une normale\n"
-"    – Deux normales\n"
+"Eqaual angles:\n"
```

Comment:
typo in English

Reply:
Will fix. Translated (with Google) as `Angles égaux :`